modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -116,8 +141,9 @@ export default class Tile3DLayer extends CompositeLayer {
   }
 
   _onTileLoad(tileHeader) {
+    const {tileset3d} = this.state;
     this.props.onTileLoad(tileHeader);
-    this._updateTileset(this.state.tileset3d);
+    this._updateTileset(tileset3d);
```

Comment:
This doesn't seem right. `_updateTileset` does not work after `activeViewports` are cleared, so you need to update it with `lastUpdatedViewports` instead here?

Reply:
Done